shared/githubDataStore.ts

```diff
@@ -0,0 +1,38 @@
+import { Octokit } from "@octokit/core";
+
+const githubOwner = "theramis";
+const githubRepo = "aucvid";
+
+const octokit = new Octokit({ auth: process.env.GITHUB_PAT });
```

Comment:
All good to have this re-used?

Reply:
Hmm I don't have a great reason but seems "safer" to not use admin auth to make a call when auth is not needed.

Reply:
Will double check I thought so too, but then saw some errors 🤔

Reply:
alg I'm not too fussed